diff --git a/lib/utils.js b/lib/utils.js
index 2b4fec4fa45e021c6296924d4191646dea8808eb..0bec2739981063f7298b6f9b5bc945715e490b3a 100644
--- a/lib/utils.js
+++ b/lib/utils.js
@@ -48,7 +48,7 @@ function is (value, expected) {
 
   /* istanbul ignore else */
   if (match) {
-    actual = match[1].trim()
+    var actual = match[1].trim()
     return actual === expected
   } else {
     return false
@@ -60,6 +60,7 @@ function hasData (string, data) {
   var value
   var keys
   var i
+  var val
 
   function empty (value) {
     return value && value.length
